### PR TITLE
Adding BytesRead to LogSource

### PR DESCRIPTION
**Pull request by DDuongNguyen**
### What does this PR do?

Getting rid of BytesRead Attribute and instead pushing it into Log Source underneath Info section

### Motivation

Creating a centralized place so logs attribute can be rendered from 1 singular place. 

### Additional Notes
Still working on how to separate the key BytesRead into 2 different words so down the road other attributes can be easily split from camel-cases accordingly. 

### Test/QA
1. Building the agent and send logs to see if the attribute get rendered correctly.
2. Checking `agent status` before and after logs have been sent
3. Checking `agent launch-gui` before and after logs have been sent

### CODE REVIEW

**Comment by gh123man** [comment]
[You have to update Line 203 here](https://github.com/DataDog/datadog-agent/pull/13939/files#diff-a9bc7ff5e7f779a6e122b4199e1440fb7606e330bf98963ef07106c864d7b196L203) as the type changed and `Load()` is no longer available on that property. So it's causing the build to fail 😄

File: pkg/logs/sources/source.go [hunk 1]
```diff
@@ -49,24 +47,26 @@ type LogSource struct {
...
-	return &LogSource{
+	source := &LogSource{
 		Name:             name,
 		Config:           cfg,
 		Status:           status.NewLogStatus(),
 		inputs:           make(map[string]bool),
 		lock:             &sync.Mutex{},
 		Messages:         config.NewMessages(),
-		BytesRead:        atomic.NewInt64(0),
+		BytesRead:        status.NewCountInfo("Byte Read"),
```
Comment (djmitche) [inline pkg/logs/sources/source.go:63]:
```suggestion
		BytesRead:        status.NewCountInfo("Bytes Read"),
```

**Review by djmitche — APPROVED** [review]
Looks OK to me, but I'll defer to agent-metrics-logs

**Review by gh123man — APPROVED** [review]
👍 
Lets update the description/QA steps. 
can remove 
> (still needs work)

probably worth adding some details: 
- check `agent status` before and after logs have been sent
- check `agent launch-gui` before and after logs have been sent

**Review by gh123man — APPROVED** [review]

**Review by hush-hush — APPROVED** [review]

**Pull request merged**